copado/main/default/classes/UserStory.cls

```diff
@@ -0,0 +1,49 @@
+public class UserStory extends DomainBuilder {
+
+    // CTORS (with 4 parameters and null ?)
```

Comment:
Remove confusing  (with 4 parameters and null ?)

Reply:
Totally, this comment was for myself, I was not sure about either overloading the constructor or just pass null values for not needed params. I think this overloadding is better